tests/loop8.py

```diff
@@ -0,0 +1,50 @@
+def nested_loop_for_for():
+    i: i32
+    j: i32
+    for i in range(2):
+        print("outer: " + str(i))
+        for j in range(10, 20):
+            print("  inner: " + str(j))
+            break
+    else:
+        print("no break in outer loop")
+
+def nested_loop_for_while():
+    i: i32
+    j: i32 = 10
+    for i in range(2):
+        print("outer: " + str(i))
+        while j < 20:
+            print("  inner: " + str(j))
+            break
+    else:
+        print("no break in outer loop")
+
+def nested_loop_while_for():
+    i: i32 = 0
+    j: i32
+    while i < 2:
+        print("outer: " + str(i))
+        i += 1
+        for j in range(10, 20):
+            print("  inner: " + str(j))
+            break
+    else:
+        print("no break in outer loop")
+
+def nested_loop_while_while():
+    i: i32 = 0
+    j: i32 = 10
+    while i < 2:
+        print("outer: " + str(i))
+        i += 1
+        while j < 20:
+            print("  inner: " + str(j))
+            break
+    else:
+        print("no break in outer loop")
+
+nested_loop_for_for()
+nested_loop_for_while()
+nested_loop_while_for()
+nested_loop_while_while()
```

Comment:
All the tests you have added in the `tests` folder generates the runtime outputs, right?

Then, I think we have to move all these `tests` into `integration_tests`, register them in `integration_tests/CMakeLists.txt` and test using `integration_tests/run_tests.py` instead of adding them to `tests.toml`

We use tests.toml only when it generates the ASR but not runtime output or LLVM.

Also, I think one or two tests is enough. Move all the functions to one or two tests.
Along with the print statements use assert statements as you did for `loop_09.py`.

Reply:
I think you forgot to add the nested tests. You can submit a new PR for it.
```py
def nested_loop_for_for():
    i: i32
    j: i32
    for i in range(2):
        print("outer: " + str(i))
        for j in range(10, 20):
            print("  inner: " + str(j))
            break
    else:
        print("no break in outer loop")

def nested_loop_for_while():
    i: i32
    j: i32 = 10
    for i in range(2):
        print("outer: " + str(i))
        while j < 20:
            print("  inner: " + str(j))
            break
    else:
        print("no break in outer loop")

def nested_loop_while_for():
    i: i32 = 0
    j: i32
    while i < 2:
        print("outer: " + str(i))
        i += 1
        for j in range(10, 20):
            print("  inner: " + str(j))
            break
    else:
        print("no break in outer loop")

def nested_loop_while_while():
    i: i32 = 0
    j: i32 = 10
    while i < 2:
        print("outer: " + str(i))
        i += 1
        while j < 20:
            print("  inner: " + str(j))
            break
    else:
        print("no break in outer loop")

nested_loop_for_for()
nested_loop_for_while()
nested_loop_while_for()
nested_loop_while_while()
```